articles/api/authentication/_change-password.md

```diff
@@ -52,9 +54,11 @@ curl --request POST \
 "We've just sent you an email to reset your password."
 ```
 
-Given a user's `email` address and a `connection`, Auth0 will send a change password email.
+Send a change password email to the user with the given email address on the given connection.
 
```

Comment:
Suggestion: Send a change password email to the user's provided email and `connection`.

Reply:
☝️ Better